src/utils/is.test.js

```diff
@@ -0,0 +1,41 @@
+import {
+  isFunction,
+  isObject
+} from './is'
+
+describe('isFunction', () => {
+  it.each([
```

Comment:
Jest's `test.each` is good magic. It creates a test for each element in this array.

https://jestjs.io/docs/api#testeachtablename-fn-timeout